Example/FinancialConnections Example/FinancialConnectionsUITests/FinancialConnectionsNetworkingUITests.swift

```diff
@@ -0,0 +1,231 @@
+//
+//  FinancialConnectionsNetworkingUITests.swift
+//  FinancialConnections Example
+//
+//  Created by Krisjanis Gaidis on 4/24/23.
+//  Copyright © 2023 Stripe, Inc. All rights reserved.
+//
+
+import XCTest
+
+final class FinancialConnectionsNetworkingUITests: XCTestCase {
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+
+        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+    }
+
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testNativeNetworkingTestMode() throws {
+        let emailAddresss = "\(UUID().uuidString)@UITestForIOS.com"
+        executeNativeNetworkingTestModeSignUpFlowTest(emailAddress: emailAddresss)
+        executeNativeNetworkingTestModeSignInFlowTest(emailAddress: emailAddresss)
```

Comment:
`UITestForIOS` is there in case someone looks at logging and wonders why there's spam

and then the idea is that we sign up a user with a random email, and then try to sign them in

step-up authentication is also tested as part of signing the user in